Packs/IntSightsV2/Integrations/IntSightsV2/IntSightsV2.py

```diff
@@ -0,0 +1,1158 @@
+                                                                          -1 * time_delta_in_days_int)
+    update_params_with_end_and_start_date(params, oldest_day_to_search_in_unix_timestamp, now_date_in_unix_timestamp)
+    del params['time-delta']
+
+
+def update_params_dict_according_to_delta_arg(params, time_delta_in_days_int):
+    if 'foundDateFrom' in params or 'foundDateTo' in params:
+        demisto.debug(
+            "ERROR in get_alerts() - can't use found-date-to or found-date-from arguments with time-delta argument")
+        return_error("Error: can't assign delta when assigned both found-date-to or found-date-from")
+    else:
+        update_params_with_delta_arg(params, time_delta_in_days_int)
+    return params
+
+
+def handle_filters(found_date_from=None):
+    """
+    Apply filters to alert list
+    """
+    args_camel_case = {
+        'alert-type': 'alertType',
+        'source-type': 'sourceType',
+        'network-type': 'networkType',
+        'source-date-from': 'sourceDateFrom',
+        'source-date-to': 'sourceDateTo',
+        'found-date-from': 'foundDateFrom',
+        'found-date-to': 'foundDateTo',
+        'is-flagged': 'isFlagged',
+        'is-closed': 'isClosed',
+        'source-ID': 'sourceId',
+        'first-seen-from': 'firstSeenFrom',
+        'first-seen-to': 'firstSeenTo',
+        'last-seen-from': 'lastSeenFrom',
+        'last-seen-to': 'lastSeenTo',
+        'value': 'iocValue',
+    }
+    params = {}
+    for key in demisto.args():
+        if demisto.getArg(key):
+            params[args_camel_case.get(key) or key] = demisto.getArg(key)
+    if demisto.getArg('time-delta'):
+        time_delta_in_days = demisto.getArg('time-delta')
+        update_params_dict_according_to_delta_arg(params, int(time_delta_in_days))
+    elif found_date_from:
+        params['foundDateFrom'] = found_date_from
+    return params
+
+
+def get_alerts_helper(params):
+    demisto.info("Executing get_alerts with params: {}".format(params))
+    response = http_request('GET', 'public/v1/data/alerts/alerts-list', params=params, json_response=True)
+
+    alerts_human_readable = []
+    alerts_context = []
+    for alert_id in response:
+        alert_human_readable, alert_context = get_alert_by_id_helper(alert_id)
+        alerts_human_readable.append(alert_human_readable)
+        alerts_context.append(alert_context)
+    return alerts_human_readable, alerts_context
+
+
+def extract_mail(replies):
+    if not replies:
+        return ''
+    mails = []
+    for reply in replies:
+        mails.append(reply.get('Email'))
+
+    return '\n'.join(mails)
+
+
+def extract_remediation(remidiations):
+    if not remidiations:
+        return ''
+    remedies = []
+    string_format = "{0} - Status: {1}"
+    for remedy in remidiations:
+        remedies.append(string_format.format(remedy.get('Value'), remedy.get('Status')))
+
+    return '\n'.join(remedies)
+
+
+def hash_identifier(hash_val):
```

Comment:
I suggest using `auto_detect_indicator_type` from `commonServerPython` instead of this function